package.json

```diff
@@ -6,7 +6,10 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "cy:open": "cypress open",
+    "test": "cypress run",
+    "start:servers:and:test": "start-test start:frontend http://localhost:3000 start:server http://localhost:3001 test"
```

Comment:
I didn't know you could pass so many parameters to `start-server-and-test`.

As per the [official docs](https://www.npmjs.com/package/start-server-and-test#options), you could do something like this:

`start-server-and-test start:frontend '3000|3001' test`

Have you tried that?